src/testinput/java/java2/util2/BitSet.java

```diff
@@ -188,7 +188,7 @@ public void flip(int fromIndex, int toIndex) {
     }
 
     int startUnitIndex = unitIndex(fromIndex);
-    long bitMask = 0;
```

Comment:
The declaration should be moved to the two (independent) places the variable is initialized.